src/main/java/Problem2/SingleLinkedList.java

```diff
@@ -1,7 +1,8 @@
 package Problem2;
 
 public class SingleLinkedList {
-    private ListNode head;
+
+    private static ListNode head;
```

Comment:
why static? 

head is specific to each single linked list. it should NOT be static?